addons/mail/static/tests/mock_server/mail_mock_server.js

```diff
@@ -20,6 +20,17 @@ import { createDocumentFragmentFromContent } from "@web/core/utils/html";
 const mockRpcRegistry = registry.category("mail.mock_rpc");
 export const DISCUSS_ACTION_ID = 104;
 
+export function bus_rpc(func) {
+    return async function (request) {
+        const { bus_rpc_uuid } = await parseRequestParams(request);
+        const result = await func.call(this, request);
+        if (bus_rpc_uuid) {
+            const [partner] = this.env["res.partner"].read(this.env.user.partner_id);
+            this.env["bus.bus"]._sendone(partner, "bus_rpc/end", bus_rpc_uuid);
+        }
+        return result;
+    };
+}
```

Comment:
I'd say we should backport the mock server part in 18.0, and wrap all discuss routes with it. Many runbot errors are affected by it, most likely recent HOOT fixes significantly contributed to the problem so the only way to avoid skipping more and more HOOT tests is to have at least this fix in HOOT tests.